.gitignore

```diff
@@ -30,3 +30,4 @@ git-src
 .cproject
 .ccache
 .vscode*
+.config
```

Comment:
Please remove this unrelated change.

Reply:
I already remove the change.
Thank you.